canonical-web-and-design/ubuntu.com
# Add raspberry pi 2 download

## Done

- Add raspberry pi 2 download pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/plex and http://0.0.0.0:8001/appliance/plex/raspberry-pi2
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that it is
    - in the menu
    - linked from the top and bottom of the appliance page
    - the download page referenced the pi 2
    - the 3 & 4 page references the pi 3 & 4
